### DISCUSSION TITLE
Breaking the ice ❄️

### DISCUSSION THREAD

**Discussion by prjctimg** [prjctimg/huetiful#59] [General]
### Hey, there!

First of all I want to thank you all for supporting my humble open source project. I've seen a generous amount of traffic on NPM (my highest peak was 800 weekly downloads!). Truly I'm humbled 🙏!!!

Throughout the journey I've stood on the shoulders of giants, without their pioneer in the field I don't think I'd have had somewhere to start from. 

This project las been inspired by libraries such as Culori, which actually provides most of the underlying functionality for general purpose colour manipulations. 

I've borrowed code from blog posts across the internet. My first utilities were slightly tweaked versions from George Francis's post  [Coloring with code ](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://tympanus.net/codrops/2021/12/07/coloring-with-code-a-programmatic-approach-to-design/&ved=2ahUKEwjWsM_FjY-CAxXjhP0HHRtzDkcQFnoECCsQAQ&usg=AOvVaw0tnXKoYhwxQl0vqch0VMCl) 

I remember at that time I was beginning to appreciate the beauty of functional programming so early versions of this project relied on lodash but I'm making an effort to reduce dependency on 3rd party libraries to keep the bundle lightweight! I was amazed at how much you could play around with simple data to produce powerful outputs. 

And of course how can I forget, chroma-js, inarguably one of the most popular libraries on the subject out there! I fell in love with its simplicity

Special thanks to [timlrx](https://github.com/timlrx) for making the Nextjs starter template that I'm using for the project's documentation site. 

And a very big thank you to the open source community for making the craft a fun, transparent and addictive experience for us newbies who still have to learn how to center a div tag !!!

**Comment by prjctimg** [comment]
Thank you, for coming aboard @arcVaishali ,@adarshswaminath @takanome-dev 

Your time and enthusiasm means a lot to the open source community?

**Reply by adarshswaminath** [reply]
Thank you for the warm welcome! I'm excited to be part of the open-source community and to collaborate with everyone on this project. I'm eager to contribute my time and skills to help make a positive impact. How can I best assist with the project? Please let me know where I can be of most help, whether it's coding, testing, documentation, or any other specific tasks. Let's work together to achieve our goals and support this amazing community!

**Reply by prjctimg** [reply]
> Thank you for the warm welcome! I'm excited to be part of the open-source community and to collaborate with everyone on this project. I'm eager to contribute my time and skills to help make a positive impact. How can I best assist with the project? Please let me know where I can be of most help, whether it's coding, testing, documentation, or any other specific tasks. Let's work together to achieve our goals and support this amazing community!

@adarshswaminath 

Great to have you aboard! Your energy and enthusiasm will make this a memorable experience for everyone.


There's more than enough chores for everyone 😂. Feel free to pick any issue that you see fit

I'm humbled by your support 🙏

**Comment by arcVaishali** [comment]
> Thank you, for coming aboard @arcVaishali ,@adarshswaminath !
> 
> Your time and enthusiasm means a lot to the open source community?

Hello **@prjctimg** ! I am more than excited to be part of this journey! Would surely love to contribute in this project.

**Reply by prjctimg** [reply]
That's great! Your enthusiasm and energy will enable us to go further!

Feel free to open/pick an issue and help make this project better for the open source community 🙏
